tensorflow/compiler/mlir/tosa/transforms/legalize_common.cc

```diff
@@ -72,7 +67,7 @@ Operation* convertPackOp(PatternRewriter& rewriter, Operation* op,
   // Check for ranked tensor type.
   if (!result_type) {
     op->emitOpError("PackOp: result type not ranked tensor");
```

Comment:
Please as followup, replace all the "emitOpError" with rewriter.notifyMatchFailure()`.

In particular in legalization the framework expects some pattern to fail to apply and this is not an error: other pattens may be able to handle it.